shared/common-adapters/avatar.render.desktop.tsx

```diff
@@ -86,11 +86,18 @@ const styles = Styles.styleSheetCreate(
         position: 'absolute',
         right: 0,
       },
-      editTeam: {
-        bottom: -2,
-        position: 'absolute',
-        right: -18,
-      },
+      editTeam: Styles.platformStyles({
```

Comment:
This doesn't look quite right with the flag off, may need to do some conditional stuff with the flag here?
![Screen Shot 2020-02-03 at 4 27 56 PM](https://user-images.githubusercontent.com/54032873/73692578-9663f180-46a2-11ea-84db-be306c550b4e.png)